plugin/zettelkasten.lua

```diff
@@ -29,18 +29,19 @@ vim.api.nvim_create_autocmd({ "BufReadCmd" }, {
 _G.zettelkasten = {
     tagfunc = require("zettelkasten").tagfunc,
     completefunc = require("zettelkasten").completefunc,
-    zknew = function()
+    zknew = function(parent_id)
         vim.cmd([[new | setlocal filetype=markdown]])
         local config = s_config.get()
         if config.notes_path ~= "" then
             vim.cmd("lcd " .. config.notes_path)
         end
 
         vim.cmd("normal ggI# New Note")
-        require("zettelkasten").set_note_id(vim.api.nvim_get_current_buf())
+        require("zettelkasten").set_note_id(vim.api.nvim_get_current_buf(), parent_id)
         vim.cmd("normal $")
     end,
     zkbrowse = function()
         vim.cmd("edit zk://browser")
     end,
+    contains = require("zettelkasten").contains,
```

Comment:
The reason these are added here is for the commands to access above. I don't think we need this one. And in fact, I should remove the other ones now that there's new API for creating commands.

Reply:
I have merged your change into this branch too.